packages/contracts/scripts/shared/helpers/loadEnv.ts

```diff
@@ -7,7 +7,7 @@ export const loadEnv = (path: string): Env => {
   dotenv.config({ path });
   const rpcUrl = process.env.RPC_URL || "http://localhost:8545";
```

Comment:
Is this updated? I believe I merged in changes that have this default to our custom gateway `https://eth.ubq.fi/v1/mainnet`

Reply:
Old version